src/main/java/uk/gov/hmcts/cmc/claimstore/documents/CitizenSealedClaimPdfService.java

```diff
@@ -0,0 +1,37 @@
+package uk.gov.hmcts.cmc.claimstore.documents;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+import uk.gov.hmcts.cmc.claimstore.config.properties.emails.StaffEmailTemplates;
+import uk.gov.hmcts.cmc.claimstore.models.Claim;
+import uk.gov.hmcts.cmc.claimstore.services.staff.content.SealedClaimContentProvider;
+import uk.gov.hmcts.reform.cmc.pdf.service.client.PDFServiceClient;
+
+import static java.util.Objects.requireNonNull;
+
+@Service
+public class CitizenSealedClaimPdfService {
+
+    private final StaffEmailTemplates emailTemplates;
```

Comment:
renamed class to EmailContentTemplates. question, why all the templates are part of staff directory?

Reply:
Because initially all templates where used to deliver something to staff users. I think email templates still might be in staff directory. Documents does not have to be there I would say because its relation with staff is much weaker.

Reply:
actioned